frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -6,184 +6,213 @@
-      "catalogId": "dev-catalog",
-      "type": "VmTemplate",
-      "provider": { "$codeRef": "createVM.catalogVMTemplateProvider" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1_API"]
     }
   },
   {
-    "type": "console.catalog/item-filter",
+    "type": "console.navigation/resource-ns",
     "properties": {
-      "catalogId": "dev-catalog",
-      "type": "Template",
-      "filter": { "$codeRef": "createVM.catalogVMTemplateFilter" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1alpha3_API"]
     }
   },
   {
-    "type": "console.redux-reducer",
+    "type": "console.page/resource/details",
     "properties": {
-      "scope": "kubevirt",
-      "reducer": { "$codeRef": "reduxReducer" }
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine"
+      },
+      "component": { "$codeRef": "VirtualMachinesDetailsPage.VirtualMachinesDetailsPage" }
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.context-provider",
+    "type": "console.navigation/href",
     "properties": {
-      "provider": { "$codeRef": "contextProvider.CDIUploadProvider" },
-      "useValueHook": { "$codeRef": "contextProvider.useCDIUploadHook" }
+      "id": "virtualmachinetemplates",
+      "section": "virtualization",
+      "perspective": "admin",
+      "name": "%kubevirt-plugin~Templates%",
+      "href": "/virtualmachinetemplates",
+      "prefixNamespaced": true
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.page/route/standalone",
+    "type": "console.page/route",
     "properties": {
-      "exact": false,
-      "path": "/standalone/k8s/ns/:ns/virtualmachineinstances/:name/console",
-      "component": { "$codeRef": "standaloneConsole.VMConsolePage" }
+      "exact": true,
+      "path": ["/k8s/ns/:ns/kubevirt.io~v1~VirtualMachine/~new/form"],
+      "component": {
+        "$codeRef": "CreateVMWizardPage.CreateVMWizardPage"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1_API"]
     }
   },
   {
-    "type": "console.pvc/create-prop",
+    "type": "console.page/route",
     "properties": {
-      "label": "%kubevirt-plugin~With Data upload form%",
-      "path": "~new/upload-form"
+      "exact": true,
+      "path": ["/k8s/ns/:ns/kubevirt.io~v1alpha3~VirtualMachine/~new/form"],
```

Comment:
We should not have explicit paths for models we support, 
an option will be to hijack toe detail page and look for name "~new"
but the dynamic plugin may have a `console.page/...` ready made for the "~new" pages, as they are widely used.
we should investigate how other resources declare the `~new` page

Reply:
@glekner @vojtechszocs do we have a dynamic plugin for "new" resource page like we have for lists and details ?

Reply:
let's do in a followup

Reply:
/~new for vm resource is done in this pr